cves/CVE-2015-6826.yml

```diff
@@ -186,8 +212,13 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    Since the vulnerability was exploitable by a specifically crafted file,
```

Comment:
How was it specifically crafted? What did users do to the file to make it trip up this error?

Reply:
Wish I could say. This is mostly extracted from the CVE that only specifies "crafted RealVideo data;" other than that and the fix commit, I could find pretty much nothing referencing or even acknowledging the vulnerability.

Reply:
I noticed that I said `RealMedia` rather than `RealVideo` here like the CVE, so I've made that fix. Thanks for leading me to that!